### PR TITLE
fix some doc bug test=document_fix

### PR DESCRIPTION
**Pull request by mrcangye**
### PR types

Bug fixes
### PR changes

Docs
### Describe

Issue：https://github.com/PaddlePaddle/docs/issues/5212

- cn docs pr: https://github.com/PaddlePaddle/docs/pull/5211

### CODE REVIEW

File: python/paddle/nn/layer/activation.py [hunk 1]
```diff
@@ -709,7 +709,7 @@ class Sigmoid(Layer):
 
     .. math::
 
-        Sigmoid(x) = \\frac{1}{1 + e^{-x}}
+        sigmoid(x) = \frac{1}{1 + e^{-x}}
```
Comment (SigureMo) [inline python/paddle/nn/layer/activation.py:714]: 这里删除转义应该不行吧，如果删除的话，在 707 行加个 `r` 自动转义吧

Comment (SigureMo) [reply python/paddle/nn/layer/activation.py:714]: 其他几个如果没有加 `r` 的加上即可

File: python/paddle/nn/functional/loss.py [hunk 2]
```diff
@@ -1049,6 +1049,7 @@ def smooth_l1_loss(input, label, reduction='mean', delta=1.0, name=None):
             label = paddle.to_tensor(label_data)
```
Comment (SigureMo) [inline python/paddle/nn/functional/loss.py]: 这里直接用 paddle 的 API 生成数据吧

File: python/paddle/nn/layer/activation.py [hunk 3]
```diff
@@ -1104,7 +1104,7 @@ class Silu(Layer):
     Silu Activation.
     .. math::
 
-        Silu(x) = \frac{x}{1 + e^{-x}}
+        silu(x) = \frac{x}{1 + e^{-x}}
 
     Parameters:
         x (Tensor): The input Tensor with data type float32, or float64.
```
Comment (SigureMo) [inline python/paddle/nn/layer/activation.py]: 没有参数 `x`，这行删掉就好了

File: python/paddle/nn/layer/loss.py [hunk 4]
```diff
@@ -1199,6 +1199,7 @@ class SmoothL1Loss(Layer):
             loss = paddle.nn.SmoothL1Loss()
```
Comment (SigureMo) [inline python/paddle/nn/layer/loss.py:1183]: 同样用 Paddle API 替代 NumPy API 生成数据

**Review by SigureMo — CHANGES_REQUESTED** [review]
部分 activation 的英文文档显示也有问题，但是写在 C++ 代码里的，~~是否需要修改周一问问梦柳吧~~ 可以试着改一下

比如 sigmoid 公式就是有问题的

image

英文文档在

https://github.com/PaddlePaddle/Paddle/blob/a635a8a5c4ec914f57058bdcccf854d620ce5f42/paddle/fluid/operators/activation_op.cc#L184-L189

和

https://github.com/PaddlePaddle/Paddle/blob/a635a8a5c4ec914f57058bdcccf854d620ce5f42/python/paddle/tensor/ops.py#L111-L124

**Review by SigureMo — DISMISSED** [review]
LGTM~~eow~~ ～

**Comment by mrcangye** [comment]
> LGTM~eow~ ～


> [mrcangye](/mrcangye) dismissed [SigureMo](/SigureMo)’s [stale review](#pullrequestreview-1119338963) via ` [7ee6825](/PaddlePaddle/Paddle/commit/7ee6825c676d3d158ad2cde56c3884067e2d6643) ` [ 6分钟前](#event-7529846573)

大佬再瞅瞅，这个dismissed [SigureMo](/SigureMo)’s [stale review]没啥影响吧，我一推送就酱紫了

File: python/paddle/nn/layer/activation.py (hunk 1 above)
Comment (mrcangye) [reply python/paddle/nn/layer/activation.py:714]: 学到了！

File: python/paddle/nn/layer/activation.py [hunk 5]
```diff
@@ -1105,7 +1105,7 @@ class Silu(Layer):
     Silu Activation.
     .. math::
```
Comment (SigureMo) [inline python/paddle/nn/layer/activation.py:1106]: 上面加一个空行吧～梦柳说的问题应该是少空行导致的

**Review by SigureMo — COMMENTED** [review]
梦柳在 docs 下回复了好几个引入 numpy 的问题，但好像只改了一个？其余几个是不用改吗？

> 大佬再瞅瞅，这个dismissed [SigureMo](https://github.com/SigureMo)’s [stale review]没啥影响吧，我一推送就酱紫了

这个没关系，Paddle 只要有新的有内容的 commit 就会 dismiss 所有已有 approval

File: python/paddle/nn/layer/activation.py (hunk 5 above)
Comment (mrcangye) [reply python/paddle/nn/layer/activation.py:1106]: 多谢提醒，辛苦大佬再审核看看有没有遗漏

File: python/paddle/nn/functional/activation.py [hunk 6]
```diff
@@ -1185,9 +1186,8 @@ def softplus(x, beta=1, threshold=20, name=None):
 
             import paddle
             import paddle.nn.functional as F
-            import numpy as np
 
-            x = paddle.to_tensor(np.array([-0.4, -0.2, 0.1, 0.3]))
+            x = paddle.to_tensor([-0.4, -0.2, 0.1, 0.3],dtype='float32')
```
Comment (SigureMo) [inline python/paddle/nn/functional/activation.py]:
```suggestion
            x = paddle.to_tensor([-0.4, -0.2, 0.1, 0.3], dtype='float32')
```

**Review by SigureMo — DISMISSED** [review]
应该没啥问题了，等梦柳过两天再预览看看吧～

**Review by SigureMo — DISMISSED** [review]

File: python/paddle/nn/functional/activation.py (hunk 6 above)
Comment (mrcangye) [reply python/paddle/nn/functional/activation.py]: 感谢大佬

File: python/paddle/nn/layer/activation.py [hunk 7]
```diff
@@ -819,9 +819,8 @@ class Softplus(Layer):
         .. code-block:: python
 
             import paddle
-            import numpy as np
 
-            x = paddle.to_tensor(np.array([-0.4, -0.2, 0.1, 0.3]))
+            x = paddle.to_tensor(np.array([-0.4, -0.2, 0.1, 0.3]),dtype='float32')
```
Comment (SigureMo) [inline python/paddle/nn/layer/activation.py]:
```suggestion
            x = paddle.to_tensor([-0.4, -0.2, 0.1, 0.3], dtype='float32')
```

**Review by Ligoml — APPROVED** [review]
good job！
LGTM

**Review by SigureMo — APPROVED** [review]
LGTM~~~
